gui/default/index.html

```diff
@@ -532,7 +532,11 @@ <h4 class="panel-title">
                       </tr>
                       <tr>
                         <th><span class="fas fa-fw fa-share-alt"></span>&nbsp;<span translate>Shared With</span></th>
-                        <td class="text-right" ng-attr-title="{{sharesFolder(folder)}}">{{sharesFolder(folder)}}</td>
+                        <td class="text-right">
+			  <span tooltip data-original-title="{{sharesFolder(folder)}}<br/>({{'Brackets indicate the remote device has not accepted this folder.' | translate}})">
```

Comment:
```suggestion
			  <span tooltip data-original-title="{{sharesFolder(folder)}}<br/>({{'Brackets indicate the remote device has not accepted this folder yet.' | translate}})">
```

I'd propose to add "yet" here, as I think it sounds more natural and also meets the expectation of the user (i.e. the folder is shared to be accepted and synced, not just to hang there). The same applies to same string in the other location.

By the way, it seems you've got a mix of spaces and tabs used for indentation.

I personally like the brackets, and it's nice to be able to get rid of the `title` attribute and use the same tooltip convention as in the rest of the GUI.

Reply:
I deliberately left out "yet", because the warning also appears when the remote has stopped sharing the folder.

Thanks for the whitespace hint, fixed for the whole file.